saved-to-the-uttermost/updated/05-chapter-4.adoc

```diff
@@ -1084,9 +1095,9 @@ footnote:[Romans 2:14]
 From all of which I argue as follows:
 
 [.syllogism]
-* In every nation, he that fears God and works righteousness is accepted.
-* But some of the heathen did fear God, and wrought righteousness by Him.
-* Therefore they were accepted.
+* __In every nation, he that fears God and works righteousness is accepted.__
+* __But some of the heathen did fear God, and wrought righteousness by Him__.
+* __Therefore they were accepted__.
```

Comment:
Can you remove the underscores for italics on these lines?  The `[.syllogism]` class takes care of them being italic as a styling concern.  If you're not seeing it correctly when outputting a pdf, make sure you've pulled in the latest changes in from the `fl` repo.  And anywhere else you're using the same class as well. Sorry I didn't make that more clear.

Reply:
I expected it to render in italics, but since it didn't I added them. I must not have the latest changes in the fl repo.